### PR TITLE
Taxonomy & SDK docs update for LocaleContext

### PR DESCRIPTION
**Pull request by sdirosa**
- [x] Taxonomy update for new properties of LocaleContext
- [x] Documentation update for LocaleContextPlugin

### CODE REVIEW

**Comment by sdirosa** [comment]
@ivarpruijn Could you check out this Taxonomy and SDK documentation update? 🙏🏼

File: docs/docs/taxonomy/reference/global-contexts/LocaleContext.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # LocaleContext
 
-A [GlobalContext](/taxonomy/reference/global-contexts/overview.md) describing the locale used by the user. E.g. an ISO 639-1 code.
+A [GlobalContext](/taxonomy/reference/global-contexts/overview.md) describing the users' language (ISO 639-1) and country (ISO 3166-1 alpha-2).
```
Comment (ivarpruijn) [inline docs/docs/taxonomy/reference/global-contexts/LocaleContext.md:3]:
```suggestion
A [GlobalContext](/taxonomy/reference/global-contexts/overview.md) describing the users' language ([ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)) and country ([ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements)).
```

**Review by ivarpruijn — APPROVED** [review]
One small suggestion, but LGTM

File: docs/docs/taxonomy/reference/global-contexts/LocaleContext.md (hunk 1 above)
Comment (sdirosa) [reply docs/docs/taxonomy/reference/global-contexts/LocaleContext.md:3]: Nice one!

**Pull request merged**